pkg/resource/podtemplatespec.go

```diff
@@ -218,6 +218,16 @@ func makePodTemplateSpec(coreClient coreset.CoreV1Interface, params *parameters.
 
 	image := os.Getenv("IMAGE")
 
+	resources := corev1.ResourceRequirements{
+		Requests: corev1.ResourceList{
+			corev1.ResourceCPU:    resource.MustParse("100m"),
+			corev1.ResourceMemory: resource.MustParse("256Mi"),
```

Comment:
@legionus @dmage i'm not sure where these defaults came from originally (i copied them from below as you can see), but they look a little low to me?  Any opinions on raising them?  Granted, an admin can now easily set them to something more appropriate as needed.

Reply:
It was a long time ago, so it's a bit outdated. And it's just a request for the scheduler, not the limit that the registry should be fit into. Maybe in 4.0 we'll be able to get better numbers from telemetry.

Reply:
ok.  we'll leave it for now.  I think Clayton said the registry was actually only using like 50megs on api.ci right now.